hadoop-ozone/ozonefs-shaded/pom.xml

```diff
@@ -102,6 +102,21 @@
             <phase>package</phase>
             <configuration>
               <skip>${maven.shade.skip}</skip>
+              <artifactSet>
+                <excludes>
+                  <!-- Leave snappy that includes native methods which cannot be relocated. -->
+                  <exclude>org.xerial.snappy:*</exclude>
```

Comment:
Classes for `org.xerial.snappy` are missing from the shaded jars.  Is that intentional?  Reading the comment I thought it was just not relocated.

Reply:
Yes, I refer to this https://github.com/viirya/hadoop/blob/7eb09762f38ac825d8614f6ecc42d041efd087bf/hadoop-client-modules/hadoop-client-api/pom.xml#L120 it said the `org.xerial.snappy` (JNI method ) should not be shaded and excluded it. 

But seem we need add it in dependency, let it as  transitive dependency to  make the downstream?

How do you think?

Reply:
Hadoop added it as dependency:

https://github.com/apache/hadoop/blob/7eb09762f38ac825d8614f6ecc42d041efd087bf/hadoop-client-modules/hadoop-client-api/pom.xml#L70-L76

But I think the idea is to include everything in the fat jar.  So I think we should skip relocation just like for bouncycastle.

Reply:
OK, let the shaded package include the `org.xerial.snappy`
This artifactSet.exclude has been removed.